### DISCUSSION TITLE
How do WinForm projects or WPF projects use this framework?

### DISCUSSION THREAD

**Discussion by 527395632** [microsoft/react-native-windows#10035] [Q&A] [answered]
Recently, this framework research has been almost completed, and the technical route has been verified, which fully meets our requirements. However, our project has been iterating for a long time and I would like to know how a project based on our previous project "WinForm Or WPF" needs to use this framework

**Comment by chrisglein** [comment, marked as answer]
This article covers adding React Native to a WPF app: https://techcommunity.microsoft.com/t5/windows-dev-appconsult/getting-started-with-react-native-for-windows/ba-p/912093

**Reply by jmurp** [reply]
Will there be an updated article or sample that details this process?  I found the material referenced out of date with the latest versions.
